libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -174,7 +174,7 @@ public override async Task<RecognizerResult> RecognizeAsync(DialogContext dc, Sc
                     // add all scores
                     foreach (var result in results)
                     {
-                        if (result.Label.Name != NoneIntent)
+                        if (results[0].Label.Name != NoneIntent || result.Label.Name != NoneIntent)
```

Comment:
May you want to change that "results[0].Label.Name != NoneIntent" into "results[0].Label.Name == NoneIntent" ?
Also, it can be refactored better moving the results[0] check/insert out of the loop?

Reply:
all different now that i understand the issue better